### DISCUSSION TITLE
ReplyKeyboardButton with call back

### DISCUSSION THREAD

**Discussion by masreplay** [aiogram/aiogram#869] [Q&A] [answered]
how to use callback like inline button with the inline keyboard in replay keyboard
is there such ReplyKeyboardButton with call back?

**Comment by evgfilim1** [comment, marked as answer]
No. ReplyKeyboard makes the client send a _message_ on button click (there are some exceptions), InlineKeyboard makes the client send _callback data_. If you want callback data, use InlineKeyboard.
